src/main/java/org/openstreetmap/atlas/utilities/scalars/Distance.java

```diff
@@ -26,30 +25,38 @@
 
     public static final String FEET_NOTATION = "'";
     public static final String INCHES_NOTATION = "\"";
+    public static final long INCHES_PER_FOOT = 12L;
+    public static final long FEET_PER_MILE = 5280;
+    public static final double METERS_PER_FOOT = 0.3048;
+    public static final double METERS_PER_KILOMETER = 1000;
+    public static final long MINUTES_PER_DEGREE = 60;
+    public static final long MILLIMETERS_PER_METER = 1000;
+    public static final long METERS_PER_NAUTICAL_MILE = 1852;
 
-    protected static final long INCHES_PER_FOOT = 12L;
-    protected static final long FEET_PER_MILE = 5280;
-    protected static final double METERS_PER_FOOT = 0.3048;
-    protected static final double METERS_PER_KILOMETER = 1000;
-    protected static final long MILLIMETERS_PER_METER = 1000;
-    protected static final long METERS_PER_NAUTICAL_MILE = 1852;
-
-    public static final Distance AVERAGE_EARTH_RADIUS = Distance.kilometers(6371);
     public static final Distance ZERO = Distance.millimeters(0);
+    public static final Distance MAXIMUM = Distance.millimeters(Long.MAX_VALUE);
+    public static final Distance AVERAGE_EARTH_RADIUS = Distance.kilometers(6371);
     public static final Distance TEN_MILES = Distance.miles(10);
     public static final Distance FIFTEEN_HUNDRED_FEET = Distance.feet(1500);
     public static final Distance ONE_METER = Distance.meters(1);
-    public static final Distance MAXIMUM = Distance.millimeters(Long.MAX_VALUE);
-
+    public static final Distance DISTANCE_PER_DEGREE_LONGITUDE_AT_EQUATOR = Distance.meters(111321);
```

Comment:
I think this one should be derived from the earth radius... This way we have only one hard coded constant.